### PR TITLE
Update runtime yamls

### PR DESCRIPTION
**Pull request by bdattoma**
Aligning our caikit+tgis runtime yaml with https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-tgis/caikit-tgis-servingruntime-grpc.yaml 

UPDATE: there will be a slight additional update to the manifest https://github.com/opendatahub-io/caikit-tgis-serving/pull/217. So I would wait merging this test PR until we include the new changes

### CODE REVIEW

**Comment by bdattoma** [comment]
PR validation: `rhods-ci-pr-test/2422`

**Review by tarukumar — APPROVED** [review]
/lgtm

**Comment by tarukumar** [comment]
you need to update the link in description properly :)

**Comment by bdattoma** [comment]
PR Validation after image URL changes:
- Sanity run for Caikit+TGIS and TGIS standalone: `rhods-ci-pr-test/2467`

**Review by FedeAlonso — APPROVED** [review]

File: ods_ci/tests/Resources/Files/llm/serving_runtimes/caikit_tgis_servingruntime_grpc.yaml [hunk 1]
```diff
@@ -10,18 +10,48 @@ spec:
...
       command: ["text-generation-launcher"]
-      args: ["--model-name=/mnt/models/artifacts/"] 
+      args: ["--model-name=/mnt/models/artifacts/"]
       env:
         - name: TRANSFORMERS_CACHE
           value: /tmp/transformers_cache
+      # resources: # configure as required
+      #   requests:
+      #     cpu: 8
+      #     memory: 16Gi
```
Comment (jstourac) [inline ods_ci/tests/Resources/Files/llm/serving_runtimes/caikit_tgis_servingruntime_grpc.yaml:22]: Are these two commented blocks for resources in these two files necessary to keep?

**Review by jstourac — APPROVED** [review]
one comment, but it's optional, lgtm

**Pull request merged**